<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mvd::MultiResolutionPyramidWidget</class>
 <widget class="QWidget" name="mvd::MultiResolutionPyramidWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="C" country="AnyCountry"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="formatLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>File format used for overview file</string>
     </property>
     <property name="text">
      <string>Format:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="formatComboBox">
     <property name="toolTip">
      <string>File format used for overview file</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="baseLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Decimation factor = Base power Level</string>
     </property>
     <property name="text">
      <string>Base:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QSpinBox" name="baseSpinBox">
     <property name="toolTip">
      <string>Decimation factor = Base power Level</string>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="0" column="4" rowspan="4">
    <widget class="QListView" name="resolutionsListView">
     <property name="toolTip">
      <string>Size list of all overview levels</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="algorithmLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Algorithm used when resampling image to compute overview level</string>
     </property>
     <property name="text">
      <string>Algorithm:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="algorithmComboBox">
     <property name="toolTip">
      <string>Algorithm used when resampling image to compute overview level</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="levelsLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Decimation factor = Base power Level</string>
     </property>
     <property name="text">
      <string>Levels:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="levelsSpinBox">
     <property name="toolTip">
      <string>Decimation factor = Base power Level</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="compressionLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Algorithm used to compress overview level data</string>
     </property>
     <property name="text">
      <string>Compression:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="compressionComboBox">
     <property name="toolTip">
      <string>Algorithm used to compress overview level data</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="sizeLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Minimal size (in pixels) of smallest overview level</string>
     </property>
     <property name="text">
      <string>Size:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="sizeSpinBox">
     <property name="toolTip">
      <string>Minimal size (in pixels) of smallest overview level</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
